content/en/blog/2021/revision-tags/index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Istio control plane canary with revisions and tags"
+description: Learn how to take more control over your Istio control-plane upgrades.
+publishdate: 2021-05-26
+attribution: "Christian Posta (Solo.io), Lin Sun (Solo.io), Sam Naser (Google)"
+keywords: [upgrades,revisions,operations,canary]
+---
+
+Like all security software, your service mesh should be kept up-to-date. The Istio community [releases new versions every quarter](/docs/releases/supported-releases/), with regular patch releases for bug fixes [and security vulnerabilities](/docs/releases/security-vulnerabilities/). The operator of a service mesh will need to upgrade the control plane and data plane components many times. You must take care when upgrading, as a mistake could affect your business traffic. Istio has many mechanisms to make it safe to perform upgrades in a controlled manner, and in Istio 1.10 we further improve this operational experience.
```

Comment:
Does it make sense to change the security vulnerability link to point to https://istio.io/latest/blog/2021/patch-tuesdays/ `Updates to how Istio security releases are handled: Patch Tuesday, embargoes, and 0-days`?  I think either link is probably good, but the blog post does mention Patch Tuesday scheduling.